### DISCUSSION TITLE
First Contribution!

### DISCUSSION THREAD

**Discussion by mariachrisochoou** [TrianguloY/URLCheck#328] [Q&A]
Hey, I am very interested in this OSS project and I was wondering if you can point me to the right direction for a good first issue? Thanks so much for your time and I can't wait to start working with you!

**Comment by TrianguloY** [comment]
Oh, thanks! 

There are some issues annotated with a "good first issue" label, that could be a good start. There may be other issues, I'll check today/tomorrow and notify here if I find another one that may be good to start.

In general, I suggest to try to find something very easy, even if it's just a couple lines! It may even be something new. Is there anything you want to add to the app? 

And also there is the option to contribute with non-code (which is also very important). It could be improving the readmes, update screenshots, translations, add something to the wiki...

Whatever you pick, if you write it here before starting, I can provide info about it, suggest ways to do it, etc.
Also, as I said, I'll try to find and label other "good first issues" in the following days.

**Comment by TrianguloY** [comment]
I've reviewed and marked a couple more issues with the "good first issue" tag. You can review them if you want.

In particular, if you are comfortable writing android apps, this could be affordable: https://github.com/TrianguloY/UrlChecker/issues/100

If you are comfortable with Github actions, this one should be easy: https://github.com/TrianguloY/UrlChecker/issues/290

And if you are more interested in no-code improvements, reviewing and updating with the latest changes the readme and/or the play store strings, or even the app descriptions strings, could be very useful.
